controllers/secret_controller.go

```diff
@@ -5,142 +5,123 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/go-logr/logr"
+	"github.com/openshift/windows-machine-config-operator/pkg/nodeconfig"
+	"github.com/openshift/windows-machine-config-operator/pkg/secrets"
+	"github.com/openshift/windows-machine-config-operator/pkg/signer"
 	"github.com/pkg/errors"
 	core "k8s.io/api/core/v1"
 	k8sapierrors "k8s.io/apimachinery/pkg/api/errors"
 	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	kubeTypes "k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/builder"
 	"sigs.k8s.io/controller-runtime/pkg/client"
-	"sigs.k8s.io/controller-runtime/pkg/client/config"
-	"sigs.k8s.io/controller-runtime/pkg/controller"
 	"sigs.k8s.io/controller-runtime/pkg/event"
 	"sigs.k8s.io/controller-runtime/pkg/handler"
-	logf "sigs.k8s.io/controller-runtime/pkg/log"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	"sigs.k8s.io/controller-runtime/pkg/predicate"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 	"sigs.k8s.io/controller-runtime/pkg/source"
-
-	"github.com/openshift/windows-machine-config-operator/pkg/cluster"
-	"github.com/openshift/windows-machine-config-operator/pkg/nodeconfig"
-	"github.com/openshift/windows-machine-config-operator/pkg/secrets"
-	"github.com/openshift/windows-machine-config-operator/pkg/signer"
 )
 
 const (
-	SecretControllerName = "secret_controller"
-	userDataSecret       = "windows-user-data"
-	userDataNamespace    = "openshift-machine-api"
+	userDataSecret    = "windows-user-data"
+	userDataNamespace = "openshift-machine-api"
 )
 
-var secretControllerLog = logf.Log.WithName(SecretControllerName)
```

Comment:
I am a little confused as to why the logger was renamed in the previous commit if this is only to removed everywhere in this commit? can this be reused in the struct?

Reply:
This needed to be renamed as both secret and wm controllers had the same named variable. That commit I had to do things like that in order to merge the controllers into a single package. If I wasnt doing multiple commits to make reviewing easier this wouldnt have happened.